extensions/amp-analytics/0.1/vendors.js

```diff
@@ -458,6 +459,16 @@ export const ANALYTICS_CONFIG = {
     },
   },
 
+  'adobeanalytics_iframePing': {
```

Comment:
Lets rename to not include `iframePing`. How about `view`?
Would like to enable us to not encode the implementation in the name.